applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -354,12 +366,53 @@ config BT_AUDIO_BROADCAST_ID_FIXED
 	  Fixed broadcast ID; 3 octets. Will only be used if BT_AUDIO_USE_BROADCAST_ID_RANDOM=n.
 	  Should only be used for debugging.
 
+config BT_AUDIO_BROADCAST_IMMEDIATE_FLAG
+	bool "Immediate rendering flag"
+	depends on TRANSPORT_BIS
+	default n
+	help
+	  Set the immediate rendering flag.
+
 config AURACAST
 	bool "Enable Auracast"
+	depends on TRANSPORT_BIS
 	default y
 	help
 	  When Auracast is enabled a Public Broadcast Announcement will be included when advertising.
 
+menu "QoS"
+
+config BT_AUDIO_PRESENTATION_DELAY_US
+	int "Presentation delay"
+	default 10000
+	help
+	  Presentation delay for the ISO link.
+
+config BT_AUDIO_MAX_TRANSPORT_LATENCY_MS
+	int "Max transport latency"
+	range 5 4000
+	default 20
+	help
+	  Max transport latency for the ISO link.
+
+config BT_AUDIO_RETRANSMITS
+	int "Number of retransmits"
+	range 0 30
+	default 2
+	help
+	  Number of retransmits for the ISO link.
+
+config BT_AUDIO_BITRATE
+	int "Bitrate for ISO stream"
+	default 96000 if BT_AUDIO_BROADCAST_RECOMMENDED || BT_AUDIO_UNICAST_RECOMMENDED
+	default 96000 if BT_AUDIO_UNICAST_CONFIGURABLE || BT_AUDIO_BROADCAST_CONFIGURABLE
```

Comment:
Should it be possible to choose bitrate for _CONFIGURABLE?

Reply:
Yes, you can still override the default by setting CONFIG_BT_AUDIO_BITRATE=80000